docs/howto/operate/new-cluster/new-cluster.md

```diff
@@ -46,16 +46,16 @@ The _minimum_ inputs this file requires are:
 
 - `subscription_id`: Azure subscription ID to create resources in.
   Should be the id, rather than display name of the project.
-- `resourcegroup_name`: The name of the Resource Group that the cluster and other resources will be deployed into.
-- `global_container_registry_name`: The name of an Azure Container Registry to use for our image.
+- `resourcegroup_name`: The name of the Resource Group to be created by terraform, where the cluster and other resources will be deployed into.
```

Comment:
I wasn't sure of this. Will the resource group be created by terraform if it doesn't exist?